### PR TITLE
[v4.2.0-rhel] vendor: bump buildah to `v1.27.1`

### PR DESCRIPTION
**Pull request by flouthoc**
Podman build: Add container GID to additional groups

[NO NEW TESTS NEEDED]
[NO TESTS NEEDED]

```release-note
None
```

### CODE REVIEW

**Review by vrothberg — APPROVED** [review]
/lgtm
/hold

**Comment by vrothberg** [comment]
@containers/podman-maintainers PTAL

**Comment by mheon** [comment]
/hold cancel
LGTM

**Comment by Mingli-Yu** [comment]
Will this commit be merged to v4.2 branch?

**Comment by mheon** [comment]
No. We are planning a new upstream release within the next few weeks, v4.3.0, which should include it and all other changes.

**Comment by Mingli-Yu** [comment]
> No. We are planning a new upstream release within the next few weeks, v4.3.0, which should include it and all other changes.

Thanks! Will wait the 4.3.0 release.

**Pull request merged**
